README.md

```diff
@@ -1,17 +1,123 @@
+This package makes it easy to send notifications using [Expo](https://docs.expo.io/push-notifications/overview/) with Laravel 8.x
 
-## Workflow for new channels
+## Contents
 
-1) Head over to the [skeleton repo](https://github.com/laravel-notification-channels/skeleton) download a ZIP copy. This is important, to ensure you start from a fresh commit history.
-2) Use find/replace to replace all of the placeholders with the correct values (package name, author name, email, etc).
-3) Implement to logic for the channel & add tests.
-4) Fork this repo, add it as a remote and push your new channel to a branch.
-5) Submit a new PR against this repo for review.
+- [Installation](#installation)
+	- [Setting up the Expo service](#setting-up-the-Expo-service)
+- [Usage](#usage)
+	- [Available Message methods](#available-message-methods)
+- [Changelog](#changelog)
+- [Testing](#testing)
+- [Security](#security)
+- [Contributing](#contributing)
+- [Credits](#credits)
+- [License](#license)
 
-Take a look at our [FAQ](http://laravel-notification-channels.com/) to see our small list of rules, to provide top-notch notification channels.
+
+## Installation
+
+Install this package with Composer:
+
+    composer require laravel-notification-channels/expo
+
+### Setting up the Expo service
+
+If you use an [Expo Access Token](https://docs.expo.io/push-notifications/sending-notifications/#additional-security) please set this in your environment.
+
+    EXPO_ACCESS_TOKEN=mysecrettoken
+
+## Usage
+Firstly you will need to define a method to retrieve your Expo push token
+````php
+class NotifiableModel extends Model {
+    // You may pass a single token
+    public function routeNotificationForExpo($notification)
+    {
+        return "ExponentPushToken[xxxxxxxxxxxxxxxxxxxxxx]"
+    }
+    
+    // Or you may return an array of tokens, for example, a user could have multiple devices.
+    public function routeNotificationForExpo($notification)
+    {
+        return $this->installations->pluck('expo_token')->toArray()
+    }
+}
+````
+
+
+````php
+<?php
+
+namespace App\Notifications;
+
+use App\Models\Message;
+use Illuminate\Bus\Queueable;
+use Illuminate\Notifications\Notification;
+use NotificationChannels\Expo\ExpoChannel;
+use NotificationChannels\Expo\ExpoMessage;
+
+class NewMessageNotification extends Notification
+{
+    use Queueable;
+
+    private Message $message;
+
+    public function __construct(Message $message)
+    {
+        $this->message = $message;
+    }
+
+    public function via($notifiable)
+    {
+        return [ExpoChannel::class];
+    }
+
+    public function toExpo($notifiable)
+    {
+        return ExpoMessage::create()
+            ->title("New Message from {$this->message->from}!")
+            ->body($this->message->text)
+            ->badge(1);
+    }
+}
+````
+
+### Available Message methods
+
+
+
+## Changelog
+
+Please see [CHANGELOG](CHANGELOG.md) for more information what has changed recently.
+
+## Testing
+
+TODO
```

Comment:
Can just leave default command here

Reply:
I thought I added that back in before pushing, after writing the tests. Whoops! Fixed :)